Investigate/fix test_proxy_arp v6 failure on backend T0

```
AssertionError: Expected packet was not received on device 0, port 4. ========== EXPECTED ========== Mask: 0000   98 03 9B 03 22 04 D4 AF  F7 0F 95 0C 86 DD 60 00   ....".........`. 0010   00 00 00 20 3A FF FE 80  00 00 00 00 00 00 D6 AF   ... :........... 0020   F7 FF FE 0F 95 0C FE 80  00 00 00 00 00 00 9A 03   ................ 0030   9B FF FE 03 22 04 88 00  93 F7 C0 00 00 00 FC 02   ...."........... 0040   10 00 00 00 00 00 00 00  00 00 00 00 00 03 02 01   ................ 0050   D4 AF F7 0F 95 0C                                  ...... mask = ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff f0 0010   00 00 ff ff ff ff ff ff  ff ff ff ff ff ff ff ff 0020   ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff 0030   ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff 0040   ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff 0050   ff ff ff ff ff ff    ========== RECEIVED ========== 0 total packets. ==============================
```

Stack trace
```
proxy_arp_enabled = True
ip_and_intf_info = ('192.168.0.2', 'fc02:1000::2', 'eth4')
ptfadapter = <tests.common.plugins.ptfadapter.ptfadapter.PtfTestAdapter testMethod=runTest>
packets_for_test = ('v6', <Ether  dst=33:33:ff:00:00:03 src=98:03:9b:03:22:04 type=0x86dd |<IPv6  nh=ICM...6ND_NS  tgt=fc02:1000::3 |<ICMPv6NDOptSrcLLAddr  lladdr=98:03:9b:03:22:04 |>>>>, <ptf.mask.Mask instance at 0x7f22fbdc3eb0>)
def test_proxy_arp(proxy_arp_enabled, ip_and_intf_info, ptfadapter, packets_for_test):
"""
Send an ARP request or neighbor solicitation (NS) to the DUT for an IP address within the subnet of the DUT's VLAN.
DUT should reply with an ARP reply or neighbor advertisement (NA) containing the DUT's own MAC
"""
pytest_require(proxy_arp_enabled, 'Proxy ARP not enabled for all VLANs')
ptf_intf_ipv4_addr, ptf_intf_ipv6_addr, ptf_intf_name = ip_and_intf_info
ptf_intf_index = int(ptf_intf_name.replace('eth', ''))
ip_version, outgoing_packet, expected_packet = packets_for_test
if ip_version == 'v4':
pytest_require(ptf_intf_ipv4_addr is not None, 'No IPv4 VLAN address configured on device')
elif ip_version == 'v6':
pytest_require(ptf_intf_ipv6_addr is not None, 'No IPv6 VLAN address configured on device')
ptfadapter.dataplane.flush()
testutils.send_packet(ptfadapter, ptf_intf_index, outgoing_packet)
>       testutils.verify_packet(ptfadapter, expected_packet, ptf_intf_index)
expected_packet = <ptf.mask.Mask instance at 0x7f22fbdc3eb0>
ip_and_intf_info = ('192.168.0.2', 'fc02:1000::2', 'eth4')
ip_version = 'v6'
outgoing_packet = <Ether  dst=33:33:ff:00:00:03 src=98:03:9b:03:22:04 type=0x86dd |<IPv6  nh=ICM...6ND_NS  tgt=fc02:1000::3 |<ICMPv6NDOptSrcLLAddr  lladdr=98:03:9b:03:22:04 |>>>>
packets_for_test = ('v6', <Ether  dst=33:33:ff:00:00:03 src=98:03:9b:03:22:04 type=0x86dd |<IPv6  nh=ICM...6ND_NS  tgt=fc02:1000::3 |<ICMPv6NDOptSrcLLAddr  lladdr=98:03:9b:03:22:04 |>>>>, <ptf.mask.Mask instance at 0x7f22fbdc3eb0>)
proxy_arp_enabled = True
ptf_intf_index = 4
ptf_intf_ipv4_addr = '192.168.0.2'
ptf_intf_ipv6_addr = 'fc02:1000::2'
ptf_intf_name = 'eth4'
ptfadapter = <tests.common.plugins.ptfadapter.ptfadapter.PtfTestAdapter testMethod=runTest>
arp/test_arp_dualtor.py:274:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/usr/lib/python2.7/dist-packages/ptf/testutils.py:2405: in verify_packet
% (device, port, result.format()))
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
self = <tests.common.plugins.ptfadapter.ptfadapter.PtfTestAdapter testMethod=runTest>
msg = 'Expected packet was not received on device 0, port 4.\n========== EXPECTED ==========\nMask:\n0000   98 03 9B 03 22 0...f ff\n0050   ff ff ff ff ff ff  \n\n========== RECEIVED ==========\n0 total packets.\n==============================\n'
def fail(self, msg=None):
"""Fail immediately, with the given message."""
>       raise self.failureException(msg)
E       AssertionError: Expected packet was not received on device 0, port 4.
E       ========== EXPECTED ==========
E       Mask:
E       0000   98 03 9B 03 22 04 D4 AF  F7 0F 95 0C 86 DD 60 00   ....".........`.
E       0010   00 00 00 20 3A FF FE 80  00 00 00 00 00 00 D6 AF   ... :...........
E       0020   F7 FF FE 0F 95 0C FE 80  00 00 00 00 00 00 9A 03   ................
E       0030   9B FF FE 03 22 04 88 00  93 F7 C0 00 00 00 FC 02   ...."...........
E       0040   10 00 00 00 00 00 00 00  00 00 00 00 00 03 02 01   ................
E       0050   D4 AF F7 0F 95 0C                                  ......
E       mask = ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff f0
E       0010   00 00 ff ff ff ff ff ff  ff ff ff ff ff ff ff ff
E       0020   ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff
E       0030   ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff
E       0040   ff ff ff ff ff ff ff ff  ff ff ff ff ff
```

--- a/tests/arp/test_arp_dualtor.py
+++ b/tests/arp/test_arp_dualtor.py
@@ -20,12 +20,13 @@ logger = logging.getLogger(__name__)
 
 
 @pytest.fixture(scope='module')
-def ip_and_intf_info(config_facts, intfs_for_test):
+def ip_and_intf_info(config_facts, intfs_for_test, ptfhost, ptfadapter):
     """
     Calculate IP addresses and interface to use for test
     """
+    ptf_ports_available_in_topo = ptfhost.host.options['variable_manager'].extra_vars.get("ifaces_map")
     _, _, intf1_index, _, = intfs_for_test
-    ptf_intf_name = "eth{}".format(intf1_index)
+    ptf_intf_name = ptf_ports_available_in_topo[intf1_index]
 
     # Calculate the IPv6 address to assign to the PTF port
     vlan_addrs = config_facts['VLAN_INTERFACE'].items()[0][1].keys()
@@ -55,7 +56,7 @@ def ip_and_intf_info(config_facts, intfs_for_test):
 
     logger.info("Using {}, {}, and PTF interface {}".format(ptf_intf_ipv4_addr, ptf_intf_ipv6_addr, ptf_intf_name))
 
-    return ptf_intf_ipv4_addr, ptf_intf_ipv6_addr, ptf_intf_name 
+    return ptf_intf_ipv4_addr, ptf_intf_ipv6_addr, ptf_intf_name, intf1_index
 
 
 @pytest.fixture
@@ -159,7 +160,7 @@ def test_arp_garp_enabled(rand_selected_dut, garp_enabled, ip_and_intf_info, int
     """
     pytest_require(garp_enabled, 'Gratuitous ARP not enabled for this device')
     duthost = rand_selected_dut
-    ptf_intf_ipv4_addr, _, _ = ip_and_intf_info
+    ptf_intf_ipv4_addr = ip_and_intf_info[0]
 
     arp_request_ip = increment_ipv4_addr(ptf_intf_ipv4_addr)
     arp_src_mac = '00:00:07:08:09:0a'
@@ -202,8 +203,7 @@ def generate_link_local_addr(mac):
 @pytest.fixture(params=['v4', 'v6'])
 def packets_for_test(request, ptfadapter, duthost, config_facts, tbinfo, ip_and_intf_info):
     ip_version = request.param
-    src_addr_v4, src_addr_v6, ptf_intf = ip_and_intf_info
-    ptf_intf_index = int(ptf_intf.replace('eth', ''))
+    src_addr_v4, src_addr_v6, _, ptf_intf_index = ip_and_intf_info
     ptf_intf_mac = ptfadapter.dataplane.get_mac(0, ptf_intf_index)
     vlans = config_facts['VLAN']
     topology = tbinfo['topo']['name']
@@ -260,8 +260,7 @@ def test_proxy_arp(proxy_arp_enabled, ip_and_intf_info, ptfadapter, packets_for_
     DUT should reply with an ARP reply or neighbor advertisement (NA) containing the DUT's own MAC
     """
     pytest_require(proxy_arp_enabled, 'Proxy ARP not enabled for all VLANs')
-    ptf_intf_ipv4_addr, ptf_intf_ipv6_addr, ptf_intf_name = ip_and_intf_info
-    ptf_intf_index = int(ptf_intf_name.replace('eth', ''))
+    ptf_intf_ipv4_addr, ptf_intf_ipv6_addr, _, ptf_intf_index = ip_and_intf_info
     ip_version, outgoing_packet, expected_packet = packets_for_test
 
     if ip_version == 'v4':
@@ -271,4 +270,4 @@ def test_proxy_arp(proxy_arp_enabled, ip_and_intf_info, ptfadapter, packets_for_
 
     ptfadapter.dataplane.flush()
     testutils.send_packet(ptfadapter, ptf_intf_index, outgoing_packet)
-    testutils.verify_packet(ptfadapter, expected_packet, ptf_intf_index)
+    testutils.verify_packet(ptfadapter, expected_packet, ptf_intf_index, timeout=10)

